project1/application.py

```diff
@@ -0,0 +1,81 @@
+import os
+from flask import Flask, session,request,render_template,flash,logging,redirect,url_for
+from flask_session import Session
+from flask_sqlalchemy import SQLAlchemy
+from datetime import datetime as dt
+from models import User,db
+
+
+app = Flask(__name__)
+
+if not os.getenv("DATABASE_URL"):
+    raise RuntimeError("DATABASE_URL is not set")
+
+app.config["SQLALCHEMY_DATABASE_URI"] = os.getenv("DATABASE_URL")
+app.config["SQLALCHEMY_TRACK_MODIFICATIONS"] = False
+app.secret_key="email"
+db.init_app(app)
+with app.app_context():
+    db.create_all()
+
+@app.route("/register", methods=["GET", "POST"])
+def register():
+
+    if (request.method == "POST"):
+        mail = request.form['name']
+        passw = request.form['pwd']
+
+        try:
+            register = User(email = mail, password = passw)
+            db.session.add(register)
+            db.session.commit()
+            User.query.all()
+            return render_template("register.html",name=mail)
+        except:
+            error="You have already registered with this email"
+            return render_template("register.html",message=error)
+    return render_template("register.html")
+
+@app.route("/login.html",methods=["GET","POST"])
+def login():
+    return render_template("login.html")
+
+@app.route("/admin")
+def admin():
+    users=User.query.all()
```

Comment:
display the latest records first, and need to render the template also

Reply:
here I order it by desc order mam